src/generic/generic-component-site-switcher/generic-component-site-switcher.twig

```diff
@@ -1,50 +1,74 @@
 {#
-  - "variant": can be '', 'header', 'footer' (default: '')
-  - "political":  {
-    - "href": target of the link (default: '')
-    - "label": label of the link (default: '')
-    - "is_active" (boolean): set active link
-
-  }
-  - "info":  {
-    - "href": target of the link (default: '')
-    - "label": label of the link (default: '')
-    - "is_active" (boolean): set active link
-  }
-  - "extra_classes" (string): extra CSS classes to be added
-  - "extra_attributes" (array): extra attributes classes (optional, format: [{ 'name': 'name_of_the_attribute', 'value': 'value_of_the_attribute'}])
+  Parameters:
+    - "variant" (string) (default: ''): can be '', 'header', 'footer'
+    - "political" (object):  {
+      - "href" (default: '')
+      - "label" (default: '')
+      - "is_active" (boolean) (default: false)
+    }
+    - "info" (object):  {
+      - "href" (default: '')
```

Comment:
There are defaults :slightly_smiling_face: 
- info: http://ec.europa.eu/info/index_en
- political: https://ec.europa.eu/commission/index_en

If not these, there's again room for a question if it's to be `#` or empty. (then please update https://github.com/ec-europa/europa-component-library/pull/665/files#diff-18ac2dca89793da06a7a241c39610899R33)

Reply:
those defaults link to the English version so if we choose to use them, we can set the labels as well

this is not something we have normalized yet: whether to use English defaults or empty defaults

@emeryro?